app/helpers/d3plus.js

```diff
@@ -77,7 +77,7 @@ export default {
     }
   },
   shapeConfig: {
-    hoverOpacity: 0.1,
+    hoverOpacity: 0.7,
```

Comment:
For the condition bars, it has to be less than 0.4, otherwise the non hovered bars look exactly the same as a hovered slightly-transparent bar. Can you a pick a value that works in that case, and then we'll use 0.7 for everything else?